hawkular-inventory-impl-tinkerpop-parent/hawkular-inventory-impl-tinkerpop/src/main/java/org/hawkular/inventory/impl/tinkerpop/TinkerpopInventory.java

```diff
@@ -74,7 +75,7 @@
                         .withProperty(IndexSpec.Property.builder()
                                 .withName(Constants.Property.__eid.name())
                                 .withType(String.class)
-                                .withUnique(true)
+//                                .withUnique(true)
```

Comment:
__eid was never supposed to be unique - only unique amongst siblings... The index would fail if I had 2 wfly feeds that would report the some resources with the same __eid (once they define their IDs to not resemble the hierarchy).